core/state/state.go

```diff
@@ -166,6 +166,7 @@ func (s *State) Update(
 	blockNum uint64,
 	update *core.StateUpdate,
 	declaredClasses map[felt.Felt]core.Class,
+	skipVerifyNewRoot bool,
```

Comment:
Please add a comment that this variable is here to satisfy the State interface

Reply:
key good catch, this is actually used, for some reason the whole file was outdated